geode-management/src/main/java/org/apache/geode/management/configuration/MemberConfig.java

```diff
@@ -92,4 +93,9 @@ public String getEndpoint() {
   public String getId() {
     return id;
   }
+
+  @Override
+  public List<String> getGroups() {
+    return Arrays.asList(getGroup());
```

Comment:
this will return a list of length 1 containing null when there is no group.  better would be to return an empty list in that case...